benchmark/python/sparse/sparse_end2end.py

```diff
@@ -169,7 +169,7 @@ def row_sparse_pull(kv, key, data, slices, weight_array, priority):
     compute_only = args.compute_only
     communication_only = args.communication_only
     if (compute_only and io_only) or (compute_only and communication_only) or (io_only and communication_only):
-        raise Exception("Only one of compute_only, io_only, communication_only can be set")
+        assert False, "Only one of compute_only, io_only, communication_only can be set"
```

Comment:
`assert compute_only + io_only + communication_only <= 1`

Reply:
Thank you for the suggestion! Changed.